metricbeat/module/vsphere/virtualmachine/data.go

```diff
@@ -78,6 +78,27 @@ func (m *MetricSet) mapEvent(data VMData) mapstr.M {
 	if len(data.triggeredAlarms) > 0 {
 		event.Put("triggered_alarms", data.triggeredAlarms)
 	}
+	if val, ok := data.PerformanceData["cpu.usage.average"]; ok {
+		event.Put("cpu.percent", val)
+	}
+	if val, ok := data.PerformanceData["disk.usage.average"]; ok {
+		event.Put("disk.average.kiloBytesPerSecond", val)
```

Comment:
Shall we have the name similar to what we have in host ?
https://github.com/elastic/beats/blob/main/metricbeat/module/vsphere/host/data.go#L84 

Also, is it different for VM ?
Is says bytes in host and kbps in vm.